project/plugins.sbt

```diff
@@ -1,2 +1,10 @@
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.7")
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
+libraryDependencies ++= Seq(
+  "com.github.pathikrit" %% "better-files" % "3.9.2",
+  "com.lihaoyi"          %% "os-lib"       % "0.10.3",
```

Comment:
is it used?

Reply:
yes, in `FormatStagedScalaFilesPlugin` on 43:5 in `getStagedScalaFiles` and on 65:5 `addToGitAllStagedFilesOnceAgain`. `better-files` is also used.